ehr_billing/resources/schemas/dbscripts/sqlserver/ehr_billing-23.001-23.002.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE ehr_billing.procedureQueryChargeIdAssoc (
+
+   rowId SERIAL NOT NULL,
+   schemaName varchar(200) NOT NULL,
+   queryName varchar(500) NOT NULL,
+   description varchar(2000) NOT NULL,
+   chargeId int NOT NULL,
+
+   container ENTITYID NOT NULL,
+   createdBy USERID,
+   created timestamp,
```

Comment:
SQLServer uses `DATETIME`, not `TIMESTAMP`

Reply:
good catch, i missed that. thanks